fop/fom_generic.c

```diff
@@ -373,12 +373,12 @@ static int fom_failure(struct m0_fom *fom)
 	}
 	if (tx->tx_state == M0_DTX_INVALID)
 		/**
-		 * Fom phase is set to M0_FOPH_TXN_COMMIT here so that when
+		 * Fom phase is set to M0_FOPH_FOL_REC_ADD here so that when
```

Comment:
What is the purpose of  adding fol records even for failure cases ?

Reply:
As comment says FOL record is not being added - `m0_fom_tick_generic()` would set the state to the next state and then it will transition to `M0_FOPH_QUEUE_REPLY`. I did this to state explicitly that there is no way to close BE tx and to not to add FOL record, because there is no state transition that allows that.

Let me add this to the comment so the next reader would know this.

Reply:
Explained.